tests/test_pipeline.py

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
What's the plan for this file?

Reply:
i have added a todo to this file as it will be replaced tomorrow with unit tests

Reply:
It is used to test that the step in the pipeline is step up correctly and can execute unit tests